src/osx/core/colour.cpp

```diff
@@ -13,127 +13,244 @@
 #include "wx/colour.h"
 
 #ifndef WX_PRECOMP
-    #include "wx/gdicmn.h"
+#include "wx/gdicmn.h"
 #endif
 
 #include "wx/osx/private.h"
 
+CGColorSpaceRef wxMacGetGenericRGBColorSpace();
+
+wxColorRefData::~wxColorRefData()
+{
+}
+
+class wxCGColorRefData : public wxColorRefData
+{
+public:
+    wxCGColorRefData(CGFloat r, CGFloat g, CGFloat b, CGFloat a = 1.0);
+
+    wxCGColorRefData(CGFloat components[4]);
+
+    wxCGColorRefData(CGColorRef);
+
+    wxCGColorRefData(const wxCGColorRefData& other);
+
+    virtual ~wxCGColorRefData()
```

Comment:
This is unnecessary. A comment like

    // default dtor is ok

would be enough. Or even nothing.